### ISSUE TITLE
Option to remove unused preprocessor commands in sim_telarray

### ISSUE THREAD

**Issue by tobiaskleiner** [gammasim/simtools#1132]
sim_telarray pfp commands include several unused directories by default such as:
`-I. -I/workdir/sim_telarray/sim_telarray/cfg -I/workdir/sim_telarray/sim_telarray/cfg/common -I/workdir/sim_telarray/sim_telarray/cfg/hess -I/workdir/sim_telarray/sim_telarray/cfg/hess2 -I/workdir/sim_telarray/sim_telarray/cfg/hess3 -I/workdir/sim_telarray/sim_telarray/cfg/hess5000 -I/workdir/sim_telarray/sim_telarray/cfg/CTA -I/workdir/sim_telarray/sim_telarray/cfg/MAGIC`

Most/all should be removable to avoid accidential configuration parameters.

**Comment by orelgueta** [comment]
Yes, I assign myself to work on it as the next step of the comparison to official productions integration test.

**Comment by orelgueta** [comment]
I think these are hardcoded in the makefile of sim_telarray:

```
# Definitions controlling conditional source compilation
DEFINES = -DCFG_INCLUDE_PATH="\".:$(PREFIX)/cfg:$(PREFIX)/$(COMMON_CFG):$(PREFIX)/$(HESS_CFG):$(PREFIX)/$(HESS2_CFG):$(PREFIX)/$(HESS3_CFG):$(PREFIX)/$(HESS5000_CFG):$(PREFIX)/$(CTA_CFG):$(PREFIX)/$(MAGIC_CFG)\"" \
          $(WHICH_DEFAULTS) -DHAVE_EVENTIO_FUNCTIONS -DNO_CFORTRAN \
	  $(EXTRA_DEFINES)
```

Maybe we can ask KB to provide an option in `build_all` not to include these locations, especially when they are not available in the process of building (we delete them before building).
